README.md

```diff
@@ -21,3 +21,18 @@ tarkatronic
 ```console
 > pip install tracktoberfest
 ```
+
+## Usage
+
+In order to use this library you will need to expose a environment variable named `GITHUB_TOKEN`
+to do this you can create a `.env` file and before running the application in your terminal run:
```

Comment:
I think it would be helpful to show what this file should contain. Specifically, a GitHub Personal Access Token in the form, `export GITHUB_TOKEN='foo'`